src/destination/selector.js

```diff
@@ -165,14 +174,14 @@ export default function destinationSelectorFactory($container, config) {
             //enable disable the action button
             this.resourceSelector.on('change', function(selected) {
                 if (selected && _.size(selected) > 0) {
-                    button.enable();
+                    self.taskCreationButton.enable();
 
                     //append the selected class URI to the task creation data
                     if (_.isPlainObject(button.config.taskCreationData)) {
-                        button.config.taskCreationData.classUri = getSelectedUri();
+                        self.taskCreationButton.config.taskCreationData.classUri = getSelectedUri();
```

Comment:
In the condition, there's an instance of `button` you forgot to change.

Reply:
true, fixing